src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -914,14 +918,14 @@ void handle_actor_task_scheduled(LocalSchedulerState *state,
   ActorID actor_id = TaskSpec_actor_id(spec);
   DCHECK(!ActorID_equal(actor_id, NIL_ACTOR_ID));
   if (state->actor_mapping.count(actor_id) == 1) {
+    DCHECK(DBClientID_equal(state->actor_mapping[actor_id].local_scheduler_id,
+                            get_db_client_id(state->db)));
+  } else {
     /* This means that an actor has been assigned to this local scheduler, and a
      * task for that actor has been received by this local scheduler, but this
      * local scheduler has not yet processed the notification about the actor
      * creation. This may be possible though should be very uncommon. If it does
      * happen, it's ok. */
-    DCHECK(DBClientID_equal(state->actor_mapping[actor_id].local_scheduler_id,
-                            get_db_client_id(state->db)));
-  } else {
     LOG_INFO(
         "handle_actor_task_scheduled called on local scheduler but the "
         "corresponding actor_map_entry is not present. This should be rare.");
```

Comment:
This might look funny, but I just moved the comment from the `if` to the `else` block. I think it was in the wrong place.